### DISCUSSION TITLE
drizzle is great, but I love prisma..

### DISCUSSION THREAD

**Discussion by Dieber** [t3-oss/create-t3-turbo#995] [General]
I know drizzle is great enough but why we can't make any choice to use primsa?
there is no way to enjoy both monorepo and prisma, is it?

**Comment by blankdreams2** [comment]
there is, you can simply install prisma and configure it inside, I have seen examples and done this by myself.

**Comment by juliusmarminge** [comment]
Search for the PR where we changed to drizzle and revert the db changes in thag PR
